grafana/provisioning/dashboards/CHT/cht_admin_api_express.json

```
@@ -0,0 +1,2738 @@
+{
+    "annotations": {
+      "list": [
+        {
+          "builtIn": 1,
+          "datasource": {
+            "type": "datasource",
+            "uid": "grafana"
+          },
+          "enable": true,
+          "hide": true,
+          "iconColor": "rgba(0, 211, 255, 1)",
+          "name": "Annotations & Alerts",
+          "type": "dashboard"
+        }
+      ]
+    },
+    "description": "Monitor metrics for CHT API node.js and express router status.",
+    "editable": true,
+    "fiscalYearStartMonth": 0,
+    "gnetId": 12230,
+    "graphTooltip": 0,
+    "id": 7,
+    "links": [],
```

Minor, but would be nice to include some dashboard links on these new dashboards similar to what the existing ones have (these just make it easier to quickly swap between dashboards).

```suggestion
    "links": [
    {
      "asDropdown": true,
      "icon": "external link",
      "includeVars": true,
      "keepTime": true,
      "tags": [],
      "targetBlank": false,
      "title": "CHT API Server",
      "tooltip": "",
      "type": "dashboards",
      "url": ""
    }
  ],
```